packages/flutter_tools/lib/src/commands/build_linux.dart

```diff
@@ -96,6 +98,16 @@ class BuildLinuxCommand extends BuildSubCommand {
       targetPlatform: targetPlatform,
       targetSysroot: stringArg('target-sysroot')!,
     );
+
+    // We don't print a size because the containing directory may contain
+    // additional files not seen by the user and because the binary is not
+    // self-contained.
```

Comment:
Consider:

```suggestion
    // We don't print a size because the output directory can contain
    // optional files not needed by the user and because the binary is not
    // self-contained.
```